infra/gcp/main.tf

```diff
@@ -470,6 +470,12 @@ module "test_gsa_secret" {
   ]
 }
 
+module "query_gsa_secret" {
```

Comment:
Good catch! Should `upload_query_jar` use this key though or was that a mistake? I.e. why not use one of the existing CI keys, or do those not have access to the query JAR bucket?

Reply:
One to ask in zulip. I'll check!